### PR TITLE
newfooter

### PR DESCRIPTION
**Pull request by jhonatanrojasbastidas**
This is the new git init about footer, this time the code is more clean and think the footer is ready for merge
mobile footer
footer pr fix

I need a little help with the responsive part.

### CODE REVIEW

File: css/home.css [hunk 1]
```diff
@@ -1,49 +1,171 @@
 /* Home page specific styles */
+@media only screen and (min-width: 360px) and (max-width: 960px) {
```
Comment (hanna446) [inline css/home.css]: Is it necessary for this line of code to be at the beginning?

File: css/home.css [hunk 2]
```diff
@@ -1,49 +1,171 @@
...
+   font-family: Roboto;
+ }
 
-.homepage-hero__promotext {
-  color: #ffffff;
-  font-size: 12px;
-  font-weight: regular;
-}
\ No newline at end of file
+ .footer__social-media:nth-last-child(1) {
```
Comment (hanna446) [inline css/home.css]: what are you doing on this line?

File: css/home.css [hunk 3]
```diff
@@ -1,49 +1,171 @@
...
+ }
+
+ .homepage-hero__headline {
+   margin-bottom: 5px;
+   color: #ffffff;
+   font-size: 36px;
+   font-weight: bold;
+ }
+
+ .homepage-hero__headline h1 {
```
Comment (hanna446) [inline css/home.css]: I think applying styles to the h1 in general, could cause problems later; you can change the name to avoid this.
Guys correct me if I'm wrong.

Comment (josemariogutierrez) [reply css/home.css]: This code is applying styles to all H1 inside the .homepage-hero__headline class, This will not affect all the H1 in the page, however it would be better if the class is added directly on the H1 element in the html to avoid using tags as selectors in CSS. Keep in mind that since is the hero section, this needs to be modified by the developer responsible of the hero.

File: css/home.css (hunk 1 above)
Comment (jhonatanrojasbastidas) [reply css/home.css]: no is not, in the css style page we use first the mobile version of the page and later we add the media query in this case the media query is worng i will fixed.

File: css/home.css (hunk 2 above)
Comment (jhonatanrojasbastidas) [reply css/home.css]: is style exclusive for the last item in the footer__social-media; this will be update in my next commit.

File: css/home.css (hunk 3 above)
Comment (jhonatanrojasbastidas) [reply css/home.css]: the hero section is not my part; i am working in the newfooter branch, i pull develop and hero is already there.

**Comment by jhonatanrojasbastidas** [comment]
mobile footer
Footer para pr

The media query was fixed, first mobile version and later desk version.
the underline was fixed is more like the zeplin version.
the code is more cleaner , i follow the bem conventions for that.

**Review by josemariogutierrez — COMMENTED** [review]
Please make pull of develop to include the latest changes into your code. Also, next time, remember always to name your branch with the convention we agreed.

**Comment by jhonatanrojasbastidas** [comment]
In my last commit is merge develop branch and newfooter branch.

File: css/style.css [hunk 4]
```diff
@@ -1,51 +1,50 @@
...
-
 /* We should use a mobile first approach having out breack point at 768px */
 /*Comodities */
 body {
   font-family: "Roboto", sans-serif;
   margin: 0;
 }
 
-.carrousel-section {
+ .carrousel-section {
```
Comment (jusk02) [inline css/style.css]: remove this extra spaces please

File: css/home.css [hunk 5]
```diff
@@ -1,18 +1,71 @@
...
 
 .homepage-hero {
+  position: absolute;
   top: 0;
   right: 0;
   bottom: 0;
   left: 0;
   width: 100%;
-  height: 100vh;
+  height: 100%;
```
Comment (jusk02) [inline css/home.css]: please restore this to `100vh;`

File: css/home.css [hunk 6]
```diff
@@ -1,18 +1,71 @@
...
+
+.footer__icons-container {
+  margin: 30px 5px;
+}
+.footer__icon {
+  margin: 7px;
+}
 
 .homepage-hero {
+  position: absolute;
```
Comment (jusk02) [inline css/home.css]: why did you add `absolute` here?

File: css/home.css [hunk 7]
```diff
@@ -1,18 +1,71 @@
...
   width: 100%;
-  height: 100vh;
+  height: 100%;
   font-family: "roboto", sans-serif;
   background: url("../assets/images/frente1.JPG") no-repeat center;
   background-size: cover;
 }
 
 .homepage-hero__layer {
+  width: 100%;
```
Comment (jusk02) [inline css/home.css]: why are you adding a `width` here?

File: css/home.css [hunk 8]
```diff
@@ -22,6 +75,7 @@
   margin-left: 5%;
   position: absolute;
   bottom: 10%;
+  width: 100%;
```
Comment (jusk02) [inline css/home.css]: why are you adding a `width` here?

File: css/home.css (hunk 7 above)
Comment (jhonatanrojasbastidas) [reply css/home.css]: i no have any idea, i just copy and paste the code from develop nothing more.

File: css/home.css (hunk 6 above)
Comment (jhonatanrojasbastidas) [reply css/home.css]: I dont write that piece of code.

File: css/home.css (hunk 8 above)
Comment (jhonatanrojasbastidas) [reply css/home.css]: that is not mine.

Comment (jusk02) [reply css/home.css]: please remove it, as gh thinks it is yours

File: css/home.css (hunk 7 above)
Comment (jusk02) [reply css/home.css]: please remove it, as gh thinks it is yours

File: css/home.css (hunk 6 above)
Comment (jusk02) [reply css/home.css]: please remove it, as gh thinks it is yours

File: css/style.css (hunk 4 above)
Comment (jhonatanrojasbastidas) [reply css/style.css]: done.

File: css/home.css (hunk 5 above)
Comment (jhonatanrojasbastidas) [reply css/home.css]: done.

File: css/home.css (hunk 6 above)
Comment (jhonatanrojasbastidas) [reply css/home.css]: done.

File: css/home.css (hunk 7 above)
Comment (jhonatanrojasbastidas) [reply css/home.css]: done.

File: css/home.css (hunk 8 above)
Comment (jhonatanrojasbastidas) [reply css/home.css]: done.

**Review by josemariogutierrez — APPROVED** [review]
Good job, please remember the next time to name your branches following the convention defined in the Workflow card in Trello.

**Review by jusk02 — APPROVED** [review]
